Tests/TuistKitAcceptanceTests/ShareAcceptanceTests.swift

```diff
@@ -29,24 +29,73 @@ final class ShareAcceptanceTests: ServerAcceptanceTestCase {
     func test_share_ios_app_with_appclip() async throws {
         try await ServiceContext.withTestingDependencies {
             try await setUpFixture(.iosAppWithAppClip)
+
+            // When: Build
             try await run(BuildCommand.self)
+
+            XCTAssertEqual(ServiceContext.current?.recordedUI(), """
```

Comment:
@tuist/community-core / @tuist/company-founders  I wonder if this is the best strategy for these tests. I like that you can see the output right there, but we are implicitly testing the format of Noora components, which means we can easily break those tests with small UI improvements in Noora components.

I'm leaning on checking that the right alerts have been scheduled for printing to make our tests easier to maintain, but I'd like to get your thoughts.

Reply:
I think these should be treated as snapshot tests. What we're missing compared to snapshot tests is to easily re-record them. We can consider using the [swift-snapshot-testing](https://github.com/pointfreeco/swift-snapshot-testing) or at least a similar approach to make it easier to re-record these snapshots when Noora's formatting changes.

I'm good with including them as-is for now, though.

Reply:
I like the idea of using swift-snapshot-testing. I used it in [Rosalind](https://github.com/tuist/rosalind) and I liked what I saw in DX.